solidity/script/avs/DeployAVS.s.sol

```diff
@@ -226,4 +227,55 @@ contract DeployAVS is Script {
             require(strategies[i] != address(0), "Strategy address is 0");
         }
     }
+
+    function addREZStrategy() external {
+        IERC20 rezToken = IERC20(0x3B50805453023a91a8bf641e279401a0b23FA6F9);
+        IStrategyFactory strategyFactory = IStrategyFactory(
+            0x5e4C39Ad7A3E881585e383dB9827EB4811f6F647
+        );
+
+        IStrategy rezStrategy = strategyFactory.deployedStrategies(rezToken);
+
+        ECDSAStakeRegistry stakeRegistry = ECDSAStakeRegistry(
+            0x272CF0BB70D3B4f79414E0823B426d2EaFd48910
+        );
+
+        // get current quorum
+        Quorum memory currentQuorum = stakeRegistry.quorum();
+        Quorum memory newQuorum;
+        newQuorum.strategies = new StrategyParams[](
+            currentQuorum.strategies.length + 1
+        );
+
+        uint256 totalStrategies = newQuorum.strategies.length;
+        uint96 baseMultiplier = uint96(10000 / totalStrategies);
+
+        uint96 remainder = 10000 % uint96(totalStrategies);
+
+        for (uint256 i = 0; i < currentQuorum.strategies.length; i++) {
+            newQuorum.strategies[i] = currentQuorum.strategies[i];
+            newQuorum.strategies[i].multiplier = baseMultiplier;
+        }
+
+        newQuorum.strategies[newQuorum.strategies.length - 1] = StrategyParams({
+            strategy: rezStrategy,
+            multiplier: baseMultiplier + remainder
+        });
+
+        console.log("New REZ Strategy: ", address(rezStrategy));
+        console.log(
+            "New REZ Strategy Multiplier: ",
+            baseMultiplier + remainder
+        );
+
+        // add strategy to quorum
+        bytes memory encodedCall = abi.encodeWithSelector(
+            stakeRegistry.updateQuorumConfig.selector,
+            newQuorum,
+            new address[](0)
```

Comment:
whats this param?

Reply:
Update operator weights, if we give an empty array, it doesn't update it.